packages/contracts-bedrock/deploy/014-FreshSystemDicator.ts

```diff
@@ -17,6 +17,7 @@ const deployFn: DeployFunction = async (hre) => {
           proxyAdmin: await getDeploymentAddress(hre, 'ProxyAdmin'),
           controller: deployer, // TODO
```

Comment:
should this be `hre.deployConfig.controller` now that this config parameter has been introduced?